resources/tracing/templates/kyma-additions/metrics-service.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/name: {{ include "jaeger-operator.fullname" . }}-jaeger-metrics
 spec:
   ports:
-  - name: metrics
+  - name: tcp-metrics
```

Comment:
Guess it's a http connection, right? Iirc correctly, having a http- prefix allows Istio to account proper http response codes in the metrics and visualize in Kiali.

Reply:
Yes, sure - TCP is simply more generic. If we want to be specific we can call it `http` also. Changing 👍

Reply:
Looks like you also have to change the port names in the ServiceMonitors to make the tests happy.